### PR TITLE
Fixes for wayland + compile mpv/postgres/ffmpeg/patch in flatpak

### PR DESCRIPTION
**Pull request by AxelTerizaki**

### CODE REVIEW

File: moe.karaokes.mugen.yml [hunk 1]
```diff
@@ -16,14 +16,758 @@ finish-args:
...
+        commit: 9069be0f15fa2c222d78a183d4aa1477d8a8f2cd
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+
+
+
+
   - name: karaokemugen
```
Comment (Dragicafit) [inline moe.karaokes.mugen.yml:771]: put karaokemugen at the end

File: moe.karaokes.mugen.yml [hunk 2]
```diff
@@ -16,14 +16,758 @@ finish-args:
...
+      - --enable-libtheora
+      - --enable-libvorbis
+      - --enable-libvpx
+      - --enable-libx264
+      - --enable-libx265
+      - --enable-libwebp
+    post-install:
+      - install -d /app/km
+      - install -d /app/km/app/
+      - install -d /app/km/app/bin
```
Comment (Dragicafit) [inline moe.karaokes.mugen.yml:319]: `install -d` is recursive

File: moe.karaokes.mugen.yml [hunk 3]
```diff
@@ -16,14 +16,758 @@ finish-args:
...
+      - -Dsdl2=enabled
+      - -Dshaderc=enabled
+      - -Dvulkan=enabled
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    post-install:
+      - install -d /app/km
+      - install -d /app/km/app/
+      - install -d /app/km/app/bin
```
Comment (Dragicafit) [inline moe.karaokes.mugen.yml:437]: `install -d` is recursive

File: moe.karaokes.mugen.yml [hunk 4]
```diff
@@ -89,7 +829,36 @@ modules:
...
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+    build-commands:
+      - ./bootstrap
+      - ./configure
+      - make
+      - install -d /app/km
+      - install -d /app/km/app/
+      - install -d /app/km/app/bin
```
Comment (Dragicafit) [inline moe.karaokes.mugen.yml:842]: `install -d` is recursive

File: moe.karaokes.mugen.yml [hunk 5]
```diff
@@ -89,7 +829,36 @@ modules:
...
+      - make 
+      - make install      
+      - cd contrib ; make ; make install
+      - ls /app/km/app/bin/postgres/share/extension/
+      - rm -Rf /app/km/app/bin/postgres/include      
+    sources:
+      - type: git
+        url: https://git.postgresql.org/git/postgresql.git
+        commit: b664e3552b800b480e2b4fadd847f8b312e00642
+        tag: REL_13_9
```
Comment (Dragicafit) [inline moe.karaokes.mugen.yml:864]: add a new line at the end

**Comment by AxelTerizaki** [comment]
Merged into v7.1.21

**Pull request closed without merging**